lnd.go

```diff
@@ -178,9 +178,11 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 	}
 
 	// Show version at startup.
-	ltndLog.Infof("Version: %s commit=%s, build=%s, logging=%s, "+
-		"debuglevel=%s", build.Version(), build.Commit,
-		build.Deployment, build.LoggingType, cfg.DebugLevel)
+	ltndLog.InfoS(ctx, "Version Info",
+		slog.String("version", build.Version()),
```

Comment:
What's the difference between `slog.String("version", build.Version()),` and just `"version", build.Version(),`?

Reply:
there is no difference in the result. The diff is that it is harder to forget a "value" for a "key" if you use these helper that provide the full `slog.Attr` instead.

Reply:
i think it can be a gut feel thing of when to use what but i think for longer log lines like this, it is nice to use Attr helpers